CONTRIBUTING.md

```diff
@@ -28,10 +28,10 @@ Start the SBT shell with `$ sbt`. The commands below assume you have a running
 sbt shell.
 
 ```sh
-> unit/test # Fast unit tests for rules, cli, core. Contains a lot
-            # of different test suites, so it's recommended to use testOnly.
-> unit/testOnly *RuleSuite # Only run tests for rules, using scalafix-testkit.
-> unit/testOnly *RuleSuite -- -z ProcedureSyntax # Only run only ProcedureSyntax unit test.
+> unitTarget2_13/test # Fast unit tests for rules, cli, core. Contains a lot
```

Comment:
we could keep `unit` for the 2.13 target by tweaking the default axis, but I feel like consistency is better than trying to elect a particular version as "main"